partner_ref_sequence/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Auto Customer Reference Number",
+    "version": "11.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "summary": "Generate Automatic customer reference number",
+    "category": "Customers",
+    "maintainer": "Open Source Integrators",
+    "website": "http://www.opensourceintegrators.com",
+    "depends": ["base"],
+    "data": [
+        "data/ir_sequence_data.xml",
+    ],
+    "installable": True,
```

Comment:
@smangukiya You are missing the development_status and the maintainers keys. Website should be https://github.com/OCA/partner-contact